BootloaderCorePkg/Stage2/Stage2.c

```diff
@@ -529,6 +530,10 @@ SecStartup (
     PlatformService->ResetSystem = ResetSystem;
   }
 
+  // SMBIOS Initialization
```

Comment:
Can we use PCD to control this feature enabling? We don't have to add empty PlatformUpdateSmbios() and etc. if SmBios is not mandatory.